internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -1305,16 +1381,18 @@ func expandAzureRmCosmosDBAccountGeoLocations(d *pluginsdk.ResourceData) ([]docu
 	return locations, nil
 }
 
-func expandAzureRmCosmosDBAccountCapabilities(d *pluginsdk.ResourceData) *[]documentdb.Capability {
+func expandAzureRmCosmosDBAccountCapabilities(d *pluginsdk.ResourceData) (*[]documentdb.Capability, map[string]interface{}) {
```

Comment:
we can remove the capabilitiesMap since this is no longer needed:

```suggestion
func expandAzureRmCosmosDBAccountCapabilities(d *pluginsdk.ResourceData) *[]documentdb.Capability {
```

Reply:
Fixed.